packages/website/plugins/tsx.js

```diff
@@ -1,33 +1,24 @@
-const { resolve } = require('path');
+module.exports = function TsxPlugin(api) {
+  api.chainWebpack(config => {
+    config.resolve.extensions.add('.tsx').add('.ts');
 
-function TsxPlugin(api) {
-    api.chainWebpack((config) => {
-        config.resolve.extensions
-            .add('.tsx')
-            .add('.ts');
+    config.resolve.alias.set('vue$', require.resolve('vue'));
 
-        config.resolve.alias
-            .set('vue$', resolve(process.cwd(), './node_modules/vue/dist/vue.esm.js'));
```

Comment:
this is the key to unblock full yarn workspace usage. You should not hardcode node_modules path.